content/en/opentelemetry/otel_terms.md

```diff
@@ -0,0 +1,36 @@
+---
+title: OpenTelemetry Terms and Concepts
+kind: documentation
+further_reading:
+    - link: 'https://opentelemetry.io/docs/concepts/'
+      tag: 'OpenTelemetry'
+      text: 'OpenTelemetry Concepts'
+    - link: 'https://opentelemetry.io/docs/concepts/glossary/'
+      tag: 'OpenTelemetry'
+      text: 'OpenTelemetry Glossary'
+    - link: 'https://docs.datadoghq.com/glossary/'
+      tag: 'Documentation'
+      text: 'Datadog Glossary'
+---
+
+This page describes essential terms and concepts for OpenTelemetry and Datadog. For a additional definitions and descriptions, see the [OpenTelemetry Glossary][6].
+
+| Concept                      | Description                                                                                                                                                                                                                                                               |
+|------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Telemetry                    | The collection of metrics, logs, and traces that provide observations about the behaviors and performance of applications and infrastructure.                                                                                                                             |
+| [OpenTelemetry Collector][1] | A vendor-agnostic agent process for collecting and exporting telemetry data emitted by many processes. It can be configured to receive, process, and export telemetry to one or multiple destinations including storage backends and analysis tools.                      |
```

Comment:
```suggestion
| [OpenTelemetry Collector][1] | A vendor-agnostic way to process, collect and export telemetry data emitted by many processes. It can be configured to receive, process, and export telemetry to one or multiple destinations including storage backends and analysis tools.                      |
```

Reply:
The collector is not actually an agent.
It can be deployed as such, but I'd rephrase it.